test/factory/peer_info.go

```diff
@@ -36,35 +29,32 @@ func NewPubkeysIdentityKeyBytes() []byte {
 func NewPubkeysProtobuf() *pb.ID_Pubkeys {
 	return &pb.ID_Pubkeys{
 		Identity: NewPubkeysIdentityKeyBytes(),
-		//Bitcoin:  []byte("AwD4y8eIx7F0bnwNmssZGi+XFqydypxuFRtA4TPyWiqJ"),
+		Bitcoin:  []byte("AwD4y8eIx7F0bnwNmssZGi+XFqydypxuFRtA4TPyWiqJ"),
```

Comment:
Pretty sure this wont validate. Is the goal to make it validate?

Reply:
Yeah, trying to get some raw factory data which works through the normal validation.

Reply:
To make it validate you will need to do base64.stdencoding.Decode()

Reply:
Looking in to this, `NewPubkeysProtobuf` is used by `NewPeerIDProtobuf` which is not used in the code at all. Should we remove both?

Reply:
Thanks for comments here. Will resolve this in a separate PR.